### DISCUSSION TITLE
help request: how add my repository to appach githab?

### DISCUSSION THREAD

**Discussion by Sheidy007** [apache/apisix#7754] [Ideas]
### Description

 I, m migrate apisix java runner from Java to  .Net C#.
I want to share it in apache comunity, how I can do it? 

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by tzssangglass** [comment]
you need to send an email to dev@apisix.apache.org and ask about this

**Reply by tokers** [reply]
@Sheidy007 Send a mail to dev@apisix.apache.org and share your idea, and achievements. Let's discuss it there.
